scrapinghub/client.py

```diff
@@ -282,21 +288,34 @@ class Spider(object):
 
     Usage::
 
-        >>> project.spiders.get('spider2')
+        >>> spider = project.spiders.get('spider1')
         <scrapinghub.client.Spider at 0x106ee3748>
         >>> spider.id
-        2
+        '1'
+        >>> spider.key
+        '123/1'
         >>> spider.name
-        spider2
+        'spider1'
     """
 
     def __init__(self, client, projectid, spiderid, spidername):
         self.projectid = projectid
-        self.id = spiderid
+        self.key = '{}/{}'.format(str(projectid), str(spiderid))
+        self.id = str(spiderid)
```

Comment:
Spider id is needed only for internal use like you mentioned, to avoid splitting the key all the time to construct queries and do some checks. Let's better make it private and remove from the docs, it makes sense to have it only for spider.